website/static/css/commentpane.css

```diff
@@ -1,42 +1,46 @@
-#commentPane {
+.commentPane {
     position: fixed;
-    z-index: 98; /* Note: Must be lower than Bootstrap modals, BlockUI */
+    z-index: 999; /* Note: Must be lower than Bootstrap modals, BlockUI */
```

Comment:
The project header nav bar and the comment pane handle are supposed to overlap each other with my UI changes. If I don't up the z-index, the handle will be completely covered by the project nav bar